common/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -120,3 +119,7 @@ databaseChangeLog:
       file: db/changelog/v2021/drop_old_coverage_table.sql
   - include:
       file: db/changelog/v2021/rework_search_event.sql
+  - include:
```

Comment:
Can't we just change the original test code? It isn't used in any of our environments so I don't think it'll interfere with liquibase. Might be worth experimenting.

Reply:
Worth a try.